Water depth processing algorithm improvements v2

leendertvanwolfswinkel opened the pull request:
This PR replaces the existing PR with the same title (merge conflicts solved)

Bugfix: Selecting DEM layer from project
Bugfix: Generating outputs for timestep 0 without moving the timestep slider
UI: Improved readability of LCD display
UI: More accurate description of what the tool does
UI: Set LCD value to 00:00 when file is loaded
Code: typo in ProcessingParameterNetcdfNumber

leendertvanwolfswinkel committed:
Water depth processing algorithm improvements

Bugfix: Selecting DEM layer from project
Bugfix: Generating outputs for timestep 0 without moving the timestep slider
UI: Improved readability of LCD display
UI: More accurate description of what the tool does
UI: Set LCD value to 00:00 when file is loaded
Code: typo in ProcessingParameterNetcdfNumber

reinout commented:
Volgens de test zit er nog ergens een typefout in:

    ./processing/threedidepth_algorithm.py:273:13: F821 undefined name 'ProcessingParamterNetcdfNumber'

leendertvanwolfswinkel committed:
Water depth minor fixes

Code: correct typo in "ProcessingParameterNetcdfNumber"
UI: Improve readability of TimeSliderCheckbox LCD display
